### DISCUSSION TITLE
經常讃美一個人好看,他真的會變得越來越好看?

### DISCUSSION THREAD

**Discussion by 111426027** [1101-GS4719-C/Python-Community-GS4719-C#1785] [Ideas]
尤其是越親近的人美聲,效果就越好,這就是美效應!這是心理學中的一種叫自我概念(Self concept)的有價值的健康和自我成長指引:讚美效應是指人們在接收到在讚美或稱讚時,會對自己產生更好的評價,從而產生一種正面的心理效應。這種效應可以使個人的自尊心得到滿足,並增強對自己的信心和積極性想像一下,你被老闆讚美了工作表現,你不自覺地微笑、感到心情愉悅、甚至更有自信心去完成更多的工作。這就是讃美效應!它就像一個心理魔法,能夠讓人們產生積極的心理反應,增強自我價值感和自信心就好像你被誇獎一樣,你會變成一個滿懷信心的超級英雄,願意去面對和克服任何挑戰。但是,過度的讚美也有可能讓你變成一個自戀狂,失去對自己的客觀評價。切記,適當的讚美可以讓你變得更好,但要保持謙虛和對自己的真實看法。

補充:讚美他人能溝通自己與他人的感情。特別是當你與他人產生隔閡時,注意和肯定他人的長處,是消除這種隔閡最有效的方式。另外,對於自己不太親近的人,恰到好處地給讚美,也會使雙方增加親近感,建立更進一步的人際關係。讚美以使人們的關係親近。同時,讚美他人還可以反過來激勵自己。所以早餐店阿姨喊你一聲帥哥,近8成的男顧客都會再次回購,這就是商業密碼!

資料來源:https://www.greenconut.com/trivia/self-concept-compliment_0305/

原來常常讚美他人還有這種功效，直到今天才知道這個冷知識! 但是通常店家叫我美女的時候我都很無感，因為那就只是他們習慣那樣叫而已，不是真的稱讚(笑

**Comment by 12Jesse24** [comment]
確實曾經有聽說過讚美會使人更漂亮，這讓我想到了之前發生在我身上的例子。
高中時候老師有出一份報告，我那時沒日沒夜地完成，最後被老師稱讚報告寫得很好，就像自己的努力有被看到一樣，之後的幾天心情都超級好，不管做什麼事情都很順，感覺自己就像充滿了能量。

**Comment by Yanxxx7979** [comment]
我覺得一定會，當人打心裡開心得時候，不只是連上會出現笑容讓人覺得有親和力
心裡感覺也會大大的影響生理，諸如大腦分泌更多多巴胺以及好的激素也會使我們的身心更加健康
而身心靈健康的時候人的氣色一定會越來越好，因此我很認同成讚會使人變漂亮這個說法

**Comment by Tomken123** [comment]
讚美甚麼，幹嘛讚美，讚美只會讓別人膨脹，要是我我一定都嘴砲優先都嘴砲優先

**Comment by uwje** [comment]
我認為還好，畢竟一直說他很醜的人也不見他變得跟醜，但認為他很帥或美的人會一直這樣，所以我認為是完全沒有影響的，但這的確是可以探討的一件事，一直詆毀他人真的會對他的狀態有所影響，所以希望不要有人去做測試

**Comment by mrweixd** [comment]
被稱讚心情就會很好，不過我認為應該是心中的期望會讓自己想要一直維持好的狀態吧

**Comment by sorn32** [comment]
我認為比起變好看，更多的是增加自信心吧。
話說有時候會覺得，自己不知道要怎麼面對別人的讚美，可以自信的回覆真的很帥氣 (´,,•ω•,,)♡

**Comment by vchien03** [comment]
會不會更好看不太好說，但增加自信是真的。當人充滿自信的時候，其實我們也不一定會去注意他「好不好看」，而是被這個人所散發出來的氣場所吸引。

**Comment by tzurou** [comment]
我覺得是自信讓人更好看，當一個人有自信的時候它散發出的氣質是會讓人想親近的，而外表我認為是因為頻繁與這個人相處而感染她的氣質覺得她變好看，大部分的人都是外表普通加上內在的自信和化妝讓人覺得他很好看。

**Comment by YOGA012** [comment]
我覺得比起讚美他人，給予親切的問候也能帶給他人正能量，以下是一些範例：
1.早安！睜開眼睛，給你一個輕輕的祝福，願它每分每秒都帶給你健康好運和幸福。
2.早安！掌握住那剎那間的永恆，樂在其中，不開心也難！祝福～順心平安快樂！
3.午安，天氣舒暢，記得多補水，中午空氣清新，祝您事事順心！！
收到這些訊息的人肯定會覺得心情愉快，一整天都充滿正能量，自然就會越來越好看。

**Comment by Mitochondrionjessica** [comment]
華人社會含蓄的風氣，常常在心裡讚美別人但不敢講出來，而謙虛的風氣導致讚美他人但對方不願坦然接受。 但我覺得適當的讚美跟接受讚美是很重要的，肯定他人的努力與付出，也接受他人的肯定。慢慢建立自信，自然就會越來越好看。

**Comment by YuanWang08** [comment]
我記得Kobe有說過一段話，內容大概是說他從來不會客套的讚美一個人，因為他覺得你沒有做得很好我還讚美你，會讓你覺得已經足夠而停滯不前，所以當他真的讚美一個人的時候，會讓那個人感覺到真正精神上的肯定與認同。不過我覺得這部分還是因人而異。

**Comment by yxin214** [comment]
以前聽過的是播好聽的音樂給植物聽、對他說好話等可以讓植物長得較好，但我一直覺得這很不科學。雖然不知道有沒有相關，但今天看了版主的文章覺得還是有那麼點邏輯在的，像是因受到稱讚而工作效率提高等。我是覺得稱讚別人其實不用額外多付出什麼，就能讓對方感到快樂，應多不吝嗇地稱讚他人。

**Comment by sandyho0307** [comment]
我覺得讚美確實是一個強大的心理工具，它不但可以提升被讚美者的心情和自信，還能增強人際關係和提升工作效率。正如你所說，即使是在關係緊張的情況下，適當的讚美可以幫助消除隔閡，增進理解和親近感。

**Comment by sunset513** [comment]
我也很認同這篇文章的觀點，根據我的觀察，不論我自己或如果收到別人的稱讚的話會十分開心。我整個人會變得更正向，臉上更有笑容，走起路更有自信，真的能讓一個人整體的難起來更好相處、更讓人喜歡！

**Comment by lilishh** [comment]
我覺得適當且真誠的讚美不僅能讓對方開心，自己心裡還會充滿正能量！也許有時候打算要誇獎別人前心中會有點說不出口，但我會為了得到更多笑容而選擇說出來，如此一來雙方都能露出笑容，還能讓關係更加融洽。

**Comment by hbh10601** [comment]
平常被讚美的時候當然會有開心、被肯定的感覺，不過低潮的時候被讚美反而會變成一把雙面刃，可能還會變得更自我懷疑，所以我認為自我肯定、往自己的目標前進才是最重要的。

**Comment by tshsu04224** [comment]
我覺得這是一定的，每個人都會需要別人給予的肯定，這些肯定累積起來才能造就一個人的自信，當一個人有自信之後，看起來就會比較容光煥發，這種自信也會使人變得比沒有自信的樣子亮眼許多，也變得更加漂亮，所以要讓一個人變漂亮，就是要不停的讚美他😎

**Comment by 710388** [comment]
是嗎？感覺好像還好欸，每次稱讚都只是禮貌吧，就算是發自內心好像也不會有那種潛移默化的感覺

**Comment by cassy0201** [comment]
被稱讚會增加個人的自信心，讓人更有底氣去面對事情!

**Comment by NcuJerryKao** [comment]
讚美效應確實是一個非常有價值的心理現象，可以帶來積極的心理效應，增強自尊心和自信心。適當的讚美不僅可以讓人感到愉悅和自信，還可以促進良好的人際關係和溝通。然而，您提到的適度讚美的重要性也是非常值得注意的。
對於過度讚美可能導致自戀或失去客觀評價的問題，這提醒我們在給予讚美時要保持平衡和真誠。確保讚美是真誠的、有根據的，並且能夠真正幫助對方提升自信和自我價值感。此外，對於自己不太親近的人，恰到好處地給予讚美可以促進更好的人際關係和溝通，同時也能激勵自己。
在商業領域中，讚美的確也有其影響力，可以建立更好的客戶關係，促進回購率等。然而，真誠的讚美和誇獎才是最有價值和有效的，因為它們是建立在真實的基礎上，能夠真正觸動人心。
感謝您分享這個有趣的心理學知識，希望這些建議能夠幫助您更好地理解和應用讚美效應，並在人際互動和溝通中獲得更多的正面影響。

**Reply by yubo0oo** [reply]
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by OHoln** [comment]
我自己認為比起讚美，對自己抱有足的信心更有效果，當一個人對自己足夠有自信，整個人散發出的氣場都完全不同

**Comment by YeeDaniel** [comment]
自信心提升：當一個人經常受到讚美時，他們的自信心會增加，感覺更好，從而可能更加注重自己的外在形象。

心理影響：積極的反饋會讓人感到開心，減少壓力。心情愉快的人往往顯得更有吸引力和活力。

**Comment by Taoyi-Chiang** [comment]
我自己滿相信相由心生這句話的欸，很多時候一個人好看與否和客觀的臉部五官比例或身材比例沒有關係，而是一個人整體給人的感覺。
如果經常讚美一個人，他就會相信這些讚美是真的，就會把自己代入這些讚美中，進而改變給人的感覺，讓人覺得越來越漂亮。
有點像我們接受他人言語的過程就是一個數學公式，而他人說了什麼就是代入一個一個的數字。
帶入好的數字就得到正面的結果，反之亦然囉！
